test_collections/sdk_tests/support/models/th_test_models.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from enum import Enum
+from pathlib import Path
+from typing import Any, Optional
+
+from pydantic import BaseModel
+
+###
+# This file declares YAML models that are used to parse the YAML Test Cases.
+###
+
+
+class THTestType(Enum):
```

Comment:
What does `TH` mean here? 

This seems to be the YAML models generalized? But does all this apply to the python tests?

Reply:
@mikaelhm The ideia www to have a common model for both YAML and Python. Maybe we should use  `SDK` instead of `TH` or even remove `TH`. What do you think?